realtime_ai_character/audio/text_to_speech/google_cloud_tts.py

```diff
@@ -0,0 +1,85 @@
+import asyncio
+import os
+import types
+import httpx
+import json
+import base64
+from google.oauth2 import service_account
+import google.auth.transport.requests
+
+from realtime_ai_character.logger import get_logger
+from realtime_ai_character.utils import Singleton
+from realtime_ai_character.audio.text_to_speech.base import TextToSpeech
+
+logger = get_logger(__name__)
+DEBUG = False
+
+config = types.SimpleNamespace(**{
+    'url': 'https://texttospeech.googleapis.com/v1/text:synthesize',
+    'headers': {
+        'Content-Type': 'application/json',
+    },
+    'data': {
+        'voice': {
+            'languageCode': 'en-US',
+            'name': 'en-US-Standard-C',
+            'ssmlGender': 'NEUTRAL'
+        },
+        'audioConfig': {
+            'audioEncoding': 'MP3'
+        }
+    },
+    'service_account_file': 'x-micron-377920-337f8e45a6b3.json'
```

Comment:
what's in service_account_file?

Reply:
this shouldn't be hardcoded. u can check how .env.example file and see how google service account was used